src/components/pages/Navbar/NavbarFeatures/SignupButton.js

```diff
@@ -5,14 +5,21 @@ import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 function SignupButton() {
   const { loginWithRedirect } = useAuth0();
 
+  const myfunction = () => {
+    window.location = '/apply';
+  };
+
   return (
     <NavBtnLink
       className="btn btn-primary btn-block"
-      onClick={() =>
-        loginWithRedirect({
-          screen_hint: 'signup',
-        })
-      }
+      // onClick={() =>
+      //   loginWithRedirect({
+      //     screen_hint: 'signup',
+      //   })
+      // }
+      onClick={() => {
+        myfunction();
+      }}
```

Comment:
Left this code commented since it will most likely be used once Auth0 has been fully implemented